cmd/tracee-ebpf/main.go

```diff
@@ -57,17 +58,35 @@ func main() {
 			// for the rest of execution, use this debug mode value
 			debug := debug.Enabled()
 
+			// Logger Setup
+			if !logger.IsSetFromEnv() {
+				logger.Init(
+					&logger.LoggerConfig{
+						Writer: os.Stderr,
+						Level: func(debug bool) logger.Level {
+
+							if debug {
```

Comment:
So this mean an environment variable will win over the debug flag. Also needs to be documented.